_im-plugin/ism/api-config-notice.md

```diff
@@ -0,0 +1,168 @@
+---
+layout: default
+title: Configuring ISM notifications settings
+parent: ISM API
+grand_parent: Index State Management
+nav_order: 10
+---
+
+## Configuring ISM notifications settings
+
+Introduced 2.8
+{: .label .label-purple }
+
+Notifications settings are configured globally, so they do not need to be configured manually for each operation. Notifications settings allow users to control how they receive notifications about index events. For example, you can choose to receive email notifications, Slack channel notifications, or both. This documentation is intended for IT administrators as a guide to configuring notifications settings.
```

Comment:
"Notifications settings are configured globally" is not very precise. If users create a LRON settings with task_id, then this settings is ad-hoc for this concrete task, and will be automatically delete after the task ends. If users create a LRON settings only with action_name, then this settings is global and persist, will apply for all operations of this action type.  And I think we should point these out to help users understand the API.